src/components/MapView/Analyser/index.tsx

```diff
@@ -215,7 +247,12 @@ function Analyser({ extent, classes }: AnalyserProps) {
 
   const clearAnalysis = () => {
     dispatch(clearAnalysisResult());
-    deactivateUniqueBoundary();
+
+    // vector hazards already use admin boundaries
+    // so no need for a unique boundary
+    if (hazardDataType !== GeometryType.Polygon) {
```

Comment:
This seems to do the opposite of the what the docstring says?

Reply:
Maybe group with what's happening on line `272`?

Reply:
And/or 278 which already has a "deactivateUniqueBoundary();"

Reply:
I believe Eric addressed this with a previous commit